### PR TITLE
Remove feature metadata in Gltf3DTileContent (for now)

### PR DESCRIPTION
**Pull request by ptrgags**
This PR targets the `3d-tiles-next` staging branch.

We're preparing to make a clean snapshot of the `3d-tiles-next` branch so we can merge existing changes into master.

Gltf3DTileContent used a prototype implementation of `EXT_feature_metadata` that transcodes it to a 3D Tiles 1.0 batch table. This is incompatible with our planned architecture, which will transcode the other way (from batch table to a 3D Tiles Next feature table).

For now, we're removing the batch table from Gltf3DTileContent. Future work will add a better implementation.

@lilleyse could you review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Source/Scene/Gltf3DTileContent.js [hunk 1]
```diff
@@ -46,15 +27,8 @@ function Gltf3DTileContent(tileset, tile, resource, gltf) {
...
-  this._batchTable = undefined;
-  this._features = undefined;
-
-  this._classificationType = tileset.vectorClassificationOnly
-    ? undefined
-    : tileset.classificationType;
 
   // Populate from gltf when available
-  this._batchIdAttributeName = undefined;
   this._diffuseAttributeOrUniformName = {};
```
Comment (lilleyse) [inline Source/Scene/Gltf3DTileContent.js:32]: This can be removed

File: Source/Scene/Gltf3DTileContent.js [hunk 2]
```diff
@@ -46,15 +27,8 @@ function Gltf3DTileContent(tileset, tile, resource, gltf) {
...
 
   this.featurePropertiesDirty = false;
```
Comment (lilleyse) [inline Source/Scene/Gltf3DTileContent.js:34]: This can be removed

File: Source/Scene/Gltf3DTileContent.js [hunk 3]
```diff
@@ -381,152 +129,67 @@ function initialize(content, gltf) {
     gltf = parseGlb(gltf);
```
Comment (lilleyse) [inline Source/Scene/Gltf3DTileContent.js:129]: This `glb` -> `gltf` parsing step is no longer required. The `gltf` object can be passed to `Model` directly which will detect whether it's a glTF or glb.

File: Source/Scene/Gltf3DTileContent.js [hunk 4]
```diff
@@ -381,152 +129,67 @@ function initialize(content, gltf) {
...
-      sphericalHarmonicCoefficients: tileset.sphericalHarmonicCoefficients,
-      specularEnvironmentMaps: tileset.specularEnvironmentMaps,
-      backFaceCulling: tileset.backFaceCulling,
-    });
-    content._model.readyPromise.then(function (model) {
-      model.activeAnimations.addAll({
-        loop: ModelAnimationLoop.REPEAT,
-      });
+  // PERFORMANCE_IDEA: patch the shader on demand, e.g., the first time show/color changes.
+  // The pick shader still needs to be patched.
```
Comment (lilleyse) [inline Source/Scene/Gltf3DTileContent.js:138]: This comment is only relevant for batch tables, can be removed

File: Source/Scene/Gltf3DTileContent.js (hunk 2 above)
Comment (ptrgags) [reply Source/Scene/Gltf3DTileContent.js:34]: No, that's part of the `Cesium3DTileContent` API

**Comment by ptrgags** [comment]
@lilleyse updated!

File: Source/Scene/Gltf3DTileContent.js (hunk 2 above)
Comment (lilleyse) [reply Source/Scene/Gltf3DTileContent.js:30]: Ah right!

**Pull request merged**